transformerlab/db.py

```diff
@@ -29,6 +31,45 @@
 # Create a configured "Session" class
 async_session = sessionmaker(async_engine, expire_on_commit=False, class_=AsyncSession)
 
+# Predefined trigger blueprints - the single source of truth for available trigger types
```

Comment:
Do not store this object in db.py. Db.py is a very tightly scoped file only used to store access to the DB but this object is a workflow specific object that should be stored somewhere else and imported in here if needed